starter_templates/crystal/app/main.cr

```diff
@@ -9,7 +9,7 @@ class YourRedisServer
     puts("Logs from your program will appear here!")
 
     # Uncomment this block to pass the first stage
-    # server = TCPServer.new("0.0.0.0", 6379, reuse_port: true)
+    # server = TCPServer.new("0.0.0.0", 6379)
```

Comment:
@ryan-gang just want to confirm, are we sure this works? i.e. have we tried running this? 

Note that the integrated tests in this repository only cover stage 1, so if there were failures that would only trigger in stage 2, the tests wouldn't highlight it.

Reply:
Ran it multiple times interleaved with another `TCPListener` from another service, bound to the same port. It seems to be working.